src/controls/IRoomControllerV2.ts

```diff
@@ -0,0 +1,380 @@
+                read: true,
+                write: true,
+                type: 'number',
+                role: 'level.temperature',
+            },
+            control.states.comfortTemperatureOffset,
+            async (name: string, value: any) => {
+                await this.setStateAck(name, value);
+                await this.updateTempTargetMinMax();
+            },
+        );
+        this.addStateChangeListener(
+            uuid + '.comfortTemperatureOffset',
+            (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+                this.sendCommand(control.uuidAction, 'setComfortModeTemp/' + newValue);
+            },
+        );
+
+        await this.updateStateObjectAsync(
+            uuid + '.tempTarget',
+            {
+                name: control.name + ': tempTarget',
+                read: true,
+                write: true,
+                type: 'number',
+                role: 'level.temperature',
+            },
+            control.states.tempTarget,
+            async (name: string, value: any) => {
+                await this.setStateAck(name, value);
+                await this.updateTempTargetMinMax();
+            },
+        );
+        this.addStateChangeListener(uuid + '.tempTarget', (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+            // Target has been set manually so start an override at that value
+            // setManualTemperature is redundant in this context as simply setting that value
+            // does not actually tell IRC to switch to manual mode.
+            // We could force whatever updates the tempTarget to then need to change mode to manual
+            // but that would be illogical as the target now shown would not actually be the target
+            // being worked with.
+            // So because of this, just start an override at the given temperature.
+            this.sendCommand(control.uuidAction, 'override/3//' + newValue);
+        });
+
+        await this.updateStateObjectAsync(
+            uuid + '.operatingMode',
+            {
+                name: control.name + ': operatingMode',
+                read: true,
+                write: true,
+                type: 'number',
+                role: 'level',
+                states: {
+                    0: 'Automatic heating/cooling',
+                    1: 'Automatic heating',
+                    2: 'Automatic cooling',
+                    3: 'Manual heating/cooling',
+                    4: 'Manual heating',
+                    5: 'Manual cooling',
+                },
+            },
+            control.states.operatingMode,
+            async (name: string, value: any) => {
+                await this.setStateAck(name, value);
+            },
+        );
+        this.addStateChangeListener(uuid + '.operatingMode', (oldValue: OldStateValue, newValue: CurrentStateValue) => {
+            this.sendCommand(control.uuidAction, 'override/' + newValue);
```

Comment:
Copy&Paste error? Should't it be `setOperatingMode/`?

Reply:
Yes, looks like copy/paste. Fixed.